modin/data_management/functions/groupby_function.py

```diff
@@ -228,6 +228,7 @@ def caller(
         numeric_only=True,
         drop=False,
         method=None,
+        default_to_pandas_func=None,
```

Comment:
This parameter was added in the recently merged #2812 and was passed and accessed via `kwargs`. Since this PR spreads `kwargs` we should explicitly add this parameter.